Corona-Warn-App/src/main/java/de/rki/coronawarnapp/worker/DiagnosisKeyRetrievalOneTimeWorker.kt

```diff
@@ -27,28 +27,40 @@ class DiagnosisKeyRetrievalOneTimeWorker(val context: Context, workerParams: Wor
      * @see RetrieveDiagnosisKeysTransaction
      */
     override suspend fun doWork(): Result {
-        Timber.d("Background job started. Run attempt: $runAttemptCount ")
+        Timber.tag(TAG).d("$id: doWork() started. Run attempt: $runAttemptCount")
```

Comment:
Nothing major but I notice you mix how variables are logged within Timber messages:
`Timber.tag(TAG).d("$id: doWork() started. Run attempt: $runAttemptCount")` (kotlin approach)
vs 
`Timber.d("sendDebugNotification(title=%s, content=%s)", title, content)` (Timber approach)

Ideally, we would align on this.

Reply:
There are a few subtle differences between both approaches. In terms of readability, I think the kotlin approach is better, but for when somethingi is not logged in a `Timber.Tree` the arguments will not be evaluated. So in cases where we log stuff like `Timber.d("A list of objects: %s", listOfObjects)` the expensive operation of expanding `listOfObjects` into a string will only happen when necessary.

Though when we improve error logging, i.e. make it available to the user, there will not be many cases where this approach actually yields any performance improvements as it will be very likely logged (and thus expanded) by a planted timber-tree anyways.
Maybe we should discuss this in the techtalk.